job-dsls/jobs/slave_image_build.groovy

```diff
@@ -52,7 +74,7 @@ def jobDefinition = job("slave-image-build") {
             string("PSI_PASSWORD", "psi-rhba-jenkins-password")
 
             // Copies the file given in the credentials to a temporary location, then sets the variable to that location.
-            file("PSI_PRIVATE_KEY", "kie-jenkins.pem")
+            file("PSI_PRIVATE_KEY", "kie-jenkis.pem")
```

Comment:
typo?

Reply:
Unfortunately, this typo is in the Jenkins master configuration and I cannot rename it there (it is typo in credential ID). I would need to recreate the credential from scratch, but I need to get out the .pem file first, it cannot be easily retrieved from Jenkins UI.